tests/test_notificationapi.py

```diff
@@ -111,3 +115,133 @@ def test_logs_and_throws_on_500(requests_mock, caplog, func, params):
     assert (
         "NotificationAPI request failed. Response: big oof 500" in caplog.text
     )
+
+
+@pytest.mark.parametrize(
+    "func,params",
+    [
+        ("create_sub_notification_id",{"notification_id": notification_id,"sub_notification_id":sub_notification_id,"title":title}),
+    ],
+)
+def test_makes_one_post_api_call_put(requests_mock, func, params):
```

Comment:
The function name does not make grammatical sense.

I suggest separating the test files for each function:

test_notificationapi_init
test_notificationapi_send
test_notificationapi_retract
test_notificationapi_createSubNotification
test_notificationapi_deleteSubNotification